plugins/infinispan-snapshot/dev.properties

```diff
@@ -0,0 +1 @@
+version.jboss.narayana=5.9.8.Final
```

Comment:
Hey @gustavolira is there any way we could get this version from this property file?: https://github.com/infinispan/infinispan/blob/5e1864da273a448f111629c2029f89c6fe5bf07f/build-configuration/pom.xml

Reply:
@alanfx actually this file should be deleted. Good catch, thanks!